components/org.wso2.carbon.identity.verification.onfido.connector/src/main/java/org/wso2/carbon/identity/verification/onfido/connector/OnfidoIdentityVerifier.java

```diff
@@ -40,169 +42,507 @@
-                    ERROR_RETRIEVING_IDV_PROVIDER.getMessage());
-        }
+        // Retrieve identity verification provider.
+        IdVProvider idVProvider = getValidatedIdVProvider(identityVerifierData, tenantId);
 
-        // Get IdV Properties that are sent through identity verification request.
-        Map<String, String> idVProperties = getIdVPropertyMap(identityVerifierData);
+        // Retrieve identity verification provider's configurations.
+        Map<String, String> idVProviderConfigProperties = getValidatedIdVConfigProperties(idVProvider);
 
-        // Get IdV Provider Config Properties.
-        Map<String, String> idVProviderConfigProperties = getIdVConfigPropertyMap(idVProvider);
-        validateIdVProviderConfigProperties(idVProviderConfigProperties);
+        // Extract the verification flow status sent via the verification request.
+        OnfidoConstants.VerificationFlowStatus verificationFlowStatus = getVerificationFlowStatus(identityVerifierData);
+        List<IdVClaim> idVClaims;
 
-        List<IdVClaim> idVClaims = new ArrayList<>();
-        switch (idVProperties.get(STATUS)) {
+        switch (verificationFlowStatus) {
             case INITIATED:
-                // Initiate Onfido verification through creating applicant and retrieving sdk token
-                idVClaims = getInitiatedVerificationResponse(userId, identityVerifierData, tenantId,
-                        idVProvider, idVProviderConfigProperties);
+                // Initiate Onfido verification through creating/updating applicant and retrieving sdk token.
+                idVClaims = initiateOnfidoVerification(userId, identityVerifierData, idVProvider,
+                        idVProviderConfigProperties, tenantId);
                 break;
             case COMPLETED:
-                // Start onfido verification through Onfido check API call.
-                idVClaims = getCheckVerificationResponse(userId, identityVerifierData, idVProvider,
+                // Complete the onfido verification flow by updating the workflow run status.
+                idVClaims = completeOnfidoVerification(userId, identityVerifierData, idVProvider,
                         idVProviderConfigProperties, tenantId);
                 break;
+            case REINITIATED:
+                 // Resends the SDK token for claims with AWAITING_INPUT status.
+                 // This reinitiates the Onfido verification flow for incomplete verifications.
+                idVClaims = reinitiateOnfidoVerification(userId, identityVerifierData, idVProvider,
+                        idVProviderConfigProperties, tenantId);
+                break;
+            default:
+                throw new IdentityVerificationClientException(ERROR_INVALID_ONFIDO_VERIFICATION_FLOW_STATUS.getCode(),
+                        ERROR_INVALID_ONFIDO_VERIFICATION_FLOW_STATUS.getMessage());
         }
         identityVerifierData.setIdVClaims(idVClaims);
         return identityVerifierData;
+
```

Comment:
```suggestion
        identityVerifierData.setIdVClaims(idVClaims);
        
        return identityVerifierData;
```

Let's add an empty line before the return statement to increase the readability. Remove the empty line after the return statement

Reply:
Resolved via https://github.com/wso2-extensions/identity-verification-onfido/pull/12/commits/95c30c53a7755ddb13b54ce67aa26e1abfb47327